server/storage/sql_models.go

```diff
@@ -17,6 +21,20 @@ func (g TUFFile) TableName() string {
 	return "tuf_files"
 }
 
+// Change defines the the fields required for an object in the changefeed
+type Change struct {
+	ID        uint `gorm:"primary_key" sql:"not null"`
+	CreatedAt time.Time
+	GUN       string `gorm:"column:gun" sql:"type:varchar(255);not null"`
+	Version   int    `sql:"not null"`
+	Sha256    string `sql:"type:varchar(64);"`
```

Comment:
Yep, we should follow Go's standard. "SHA" is technically an acronym so it shold either be all caps, `SHA`, or all lower case, `sha`, depending on whether it needs to be exported or not. I'll fix it for this new struct and file an issue for us to go review other places.